server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/Tablet.java

```diff
@@ -178,6 +178,8 @@ enum CompactionState {
 
   private final int logId;
 
+  private volatile long lastAccessTime = 0;
```

Comment:
Can this be set at creation?

```suggestion
  private volatile long lastAccessTime = System.nanoTime();
```

Reply:
Implemented in a7361ea